other-docs/guides/migrating-from-wordpress.md

```diff
@@ -0,0 +1,150 @@
+
+- `smart-media`
+- `gaussholder`
+- `aws-rekognition`
+- `two-factor`
+- `stream`
+- `wp-simple-saml`
+- `delegated-oauth`
+- `elasticpress`
+- `hm-redirects`
+- `msm-sitemap`
+- `wp-seo`
+- `amp`
+- `facebook-instant-articles-wp`
+- `meta-tags`
+- `query-monitor`
+- `hm-gtm`
+- `workflows`
+- `wp-user-signups`
+
+### Move wp-config.php
+
+Installing HM Platform will replace your `wp-config.php` so you should back it up if you have application level configuration in your `wp-config.php`.
+
+### Installing HM Platform
+
+Now you have your composer project up and running, it’s time to add HM Platform to the project. Do so by running `composer require humanmade/platform`.
+
+Once HM Platform has been installed, you should see the `wordpress` directory back in the project root, a new `wp-config.php` and `index.php`. You should now also see these 3 paths added to your `.gitignore`.
+
+### Restore site configuration
+
+If you have any custom configuration in your old `wp-config.php` (such as custom PHP constants, etc. You will need to put them into a new file outside of `wp-config.php` (as changes to this file are not allowed). It’s possible your project already has the configuration split out into a `.config` directory, but if not, create a file at `.config/load.php`.
+
+Only copy across constants that your custom code actually needs. There should be no database constants, or any other WordPress type constants. If you have these already in your `.config` directory, you should delete those.
+
+To make `.config/load.php` actually be included and executed, you have to add it as a Composer autoload file. To do so, add an `autoload` section to your `composer.json` if you don’t already have one, and add `.config/load.php` as an autoload files.
+
+```json
+	"autoload": {
+		"files": [
+			".config/load.php"
+		]
+	},
+```
+
+You’ll need to run `composer dump-autoload` after doing this to make sure it’s actually loaded.
+
+### Migrate $hm_platform options (when applicable)
+
+In your old `wp-config.php` you’ll see there is a `global $hm_platform` that sets options for hm-platform. Only migrate anything that you specifically need to, as most likely HM Platform will have better defaults. In rare cases though, things will be disabled for good reason. In HM Platform, most of the same settings are supported, but it’s now done via the `composer.json` for configuration (as is all HM Platform configuration). You should be familiar with [HM Platform configuration](docs://getting-started/configuration.md) before continuing. `$hm_platform` options should go in the `platform.modules.cloud` section of the `extra` block in the `composer.json`.
+
+```json
+"extra": {
```

Comment:
This might get rendered showing errors, github itself at least highlights syntax errors